util/codec/bytes.go

```diff
@@ -116,8 +119,8 @@ func decodeBytes(b []byte, reverse bool) ([]byte, []byte, error) {
 
 // DecodeBytes decodes bytes which is encoded by EncodeBytes before,
 // returns the leftover bytes and decoded value if no error.
-func DecodeBytes(b []byte) ([]byte, []byte, error) {
-	return decodeBytes(b, false)
+func DecodeBytes(b []byte, data []byte) ([]byte, []byte, error) {
```

Comment:
I think `buf` is a better name.
And please update the comment of the newly added argument.